src/test/java/jp/co/future/uroborosql/SqlAgentTest.java

```diff
@@ -392,6 +392,31 @@ public void testQueryMapResuletSetConverter() throws Exception {
 	}
 
 	/**
+	 * クエリ実行処理のテストケース(Fluent API)。
+	 */
+	@Test
+	public void testQueryFluentLamdaAndUpdate() throws Exception {
```

Comment:
The method name `testQueryFluentLamdaAndUpdate` is incorrect. You should do as follows.

incollect: `testQueryFluentLamdaAndUpdate`  
collect　:   `testQueryFluentLambdaAndUpdate`

There are other methods that are `Lamda`, so please correct them as well.